tensorflow/compiler/xla/service/gpu/cusolver_context.cc

```diff
@@ -19,6 +19,9 @@ limitations under the License.
 #include <complex>
 #include <utility>
 
+#if TENSORFLOW_USE_ROCM
```

Comment:
I don't think this is necessary because it is in cusolver_context.h.

Reply:
Removed.